src/main/java/com/exadel/discount/model/dto/vendor/VendorDTO.java

```diff
@@ -16,6 +16,7 @@
     private UUID id;
     private String name;
     private String description;
+    private String image;
```

Comment:
Maybe vendorImage will be more explicite?

Reply:
No 😿